.github/updatecli/updatecli.d/versions.yml

```diff
@@ -73,18 +73,18 @@ sources:
       versionfilter:
         kind: latest
 
-  # latest-edot-cf-gcp-version:
-  #   name: Get latest release version for the edot-cf-gcp
-  #   kind: githubrelease
-  #   transformers:
-  #     - trimprefix: v
-  #   spec:
-  #     owner: elastic
-  #     repository: edot-cloud-forwarder-gcp
-  #     token: '{{ requiredEnv "GITHUB_TOKEN" }}'
-  #     username: '{{ requiredEnv "GITHUB_ACTOR" }}'
-  #     versionfilter:
-  #       kind: latest
+  latest-edot-cf-gcp-version:
```

Comment:
Can we rename this do `latest-docker-edot-cf-gcp-version`?

And add a new one called `latest-terraform-google-edot-cf-version`? That comes from this [repository](https://github.com/elastic/terraform-google-edot-cloud-forwarder).

Reply:
@constanca-m Both versions will always be in sync? If that's the case, I would just fetch one, unless we expect them to diverge.

In any case, let's wait till the other repo is public before merging this PR. I'll now draft the changes for the other repo.

Reply:
They won't be in sync, they are different things: one is for docker release, go code; and the other is for terraform module.

Both are private at the moment, and ECF GCP repository will always be private